### ISSUE TITLE
Question about eval_class

### ISSUE THREAD

**Issue by LinHungShi** [prclibo/kitti_eval#5]
Hi, thanks for your hard work on implementing the evaluation code. I am looking into the code and have a hard time connecting it to the AP formula in the paper. Most confusion comes from eval_class, which calculates the AP of a single class. Since it's hard for me to organize my questions into one paragraph, I list all of them in the below:

1. Why use detection scores as recall? More specifically, why you use scores as threshold when computing statistics? As far as I know, there's no threshold when computing average precision, which only involves calculating precisions for discretized recalls.

2. Following the previous question, the discretized recalls are 0, 0.1,....,1 as described in the paper. Where do you use this information in eval_class

3.What does N_SAMPLE_PTS mean, and how do you come up with 41?

4. Why we need getThresholds and how does it work?

To sum up, I don't quite understand how it connects to the AP formula, and would appreciate if you can help me out!!!
